nuget.config

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="dotnet-experimental" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-experimental/nuget/v3/index.json" />
```

Comment:
You're using experimental packages. If Native AOT relies on experimental packages, I don't think it's wise to include it on a project that demands absolute stability.

Reply:
"absolute stability" is a knee-jerk phrase to use. we're willing to test things (i.e. prerelease realm packages), *as long as* the consequences are properly understood and conveyed. which, in this case, they're not.